src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabView.cs

```diff
@@ -869,6 +869,13 @@ void OnTabContentPresenterLosingFocus(object sender, LosingFocusEventArgs args)
 
 						tabContentPresenter.LosingFocus += OnTabContentPresenterLosingFocus;
 
+#if IS_UNO
+						// TODO: Uno specific - issue #4894 - in UWP the ContentPresenter does not become
+						// the parent of the Content. In Uno it does, so we need to make sure
+						// the inherited DataContext will match the TabViewItem.
+						tabContentPresenter.DataContext = tvi.DataContext;
```

Comment:
Is this possible to test in a runtime test? It'll help when we'll work on ContentControl alignments

Reply:
You mean to create unit tests that will be disabled for now, testing the correct behavior in general, or to test this workaround?

Reply:
We need to validate the scenario does not regress, by validating this workaround.

Reply:
Done 👍